packages/ds/src/admin/form/InputText.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { HiOutlineExclamation } from "react-icons/hi";
+
+type OwnProps = {
+  disabled?: boolean;
+  active?: boolean;
+  error?: string;
+  name: string;
+  value?: string;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+};
+
+function classNames(...classes: any[]) {
+  return classes.filter(Boolean).join(" ");
+}
+
+export default function InputText({
+  disabled,
+  active,
+  error,
+  name,
+  value,
+  onChange,
```

Comment:
Pareil ici pour l'ordre et la clareté.

```suggestion
  name,
  onChange,
  disabled,
  active,
  error,
  value,
```